#config file for adding the strands specific recovery berhaviours to the monitored navigation state machine
nav_recovery:
    package: strands_recovery_behaviours
    recovery_file: recover_nav
    recovery_class: RecoverNav
monitor_recovery_pairs:
    -   name: bumper
        package: strands_recovery_behaviours
        monitor_file: monitor_bumper
        monitor_class: MonitorBumper
        recovery_file: recover_bumper
        recovery_class: RecoverBumper
    -   name: stuck_on_carpet
        package: strands_recovery_behaviours
        monitor_file: monitor_stuck_on_carpet
        monitor_class: MonitorStuckOnCarpet
        recovery_file: recover_stuck_on_carpet_no_help
        recovery_class: RecoverStuckOnCarpetNoHelp
    -   name: pause_nav
        package: strands_recovery_behaviours
        monitor_file: monitor_pause
        monitor_class: MonitorPause
        recovery_file: pause_state
        recovery_class: PauseState
human_help:
    -   name: speech
        package: strands_human_help
        helper_file: help_speech
        helper_class: HelpSpeech
    -   name: screen
        package: strands_human_help
        helper_file: help_screen
        helper_class: HelpScreen